src/streams/transform.js

```diff
@@ -1,3 +1,12 @@
+import {Transform} from 'stream'
+
+const reverseStream = new Transform({transform(chunk, encoding, callback) {
+        const stringedChunk = chunk.toString();
+        const reversedChunk = stringedChunk.split('').reverse().join('');
+        console.log(reversedChunk)
```

Comment:
```suggestion
        callback(null, reversedChunk)
```
А иначе этот стрим не передаёт данные дальше по цепочке, а только сам их выводит